README.md

```diff
@@ -25,6 +26,11 @@
 - Flexible build pipeline
 - Cross platform builds
 
+## Installing
+`npm install nexe -g`
```

Comment:
Isn't this only needed if you've just setup an install of `npm`? It'd probably be nice if `nexe` would release binary versions of itself, though 😄

Reply:
I don't understand, ha.. 

To use nexe you have to install it at some point, right?  There's a binary under releases.